examples/device/cdc_msc/src/msc_disk.c

```diff
@@ -192,6 +192,9 @@ int32_t tud_msc_read10_cb(uint8_t lun, uint32_t lba, uint32_t offset, void* buff
   // out of ramdisk
   if ( lba >= DISK_BLOCK_NUM ) return -1;
 
+  // Check for overflow of offset + bufsize
+  if ( offset + bufsize >= DISK_BLOCK_SIZE ) return -1;
```

Comment:
Logic is not correct, offset = 0 and bufsize = 512 is correct value. This cause hardware test (hil) to fail on all cdc_msc test. We will need to revert this PR